stylesheets/_layout.settings.scss

```diff
@@ -584,19 +585,26 @@
  * Sticky sidebar
  */
 
-@media screen and (min-width: 992px) {
-    .tab__inner.is-sticky .tab__content {
-        left: calc(370px + 1%);
-        max-width: calc(100% - 370px);
-        position: relative;
-    }
+@include respond-min($screen-medium) {
+    .tab__inner.is-sticky {
+        min-height: 100vh;
+        padding-bottom: 0;
+        margin-bottom: -37px;
```

Comment:
Magic number needs explaining in a comment, presume it's to cater for footer?

Reply:
It is indeed to cater for the footer height. Do we have a variable for this?

Reply:
Have added (and wired in) a $footer-height token which does the same calculation of line height + padding + border